x-pack/plugins/licensing/public/plugin.ts

```diff
@@ -86,12 +83,19 @@ export class LicensingPlugin implements Plugin<LicensingPluginSetup> {
         // Prevent reusing stale license if the fetch operation fails
         this.removeSaved();
       }
+
+      if (license.status === 'expired' && !this.isLicenseExpirationBannerShown && this.coreStart) {
+        this.isLicenseExpirationBannerShown = true;
```

Comment:
I replicated LP behavior when a banner was shown only once. Shall we reset this flag if a license was updated with `active` status?

Reply:
Unlikely someone would have the same browsing session open during two expirations of the license, but that does seem preferable if it's easy. Otherwise, I wouldn't worry about it.

Reply:
will update in a follow up